src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -464,7 +467,7 @@ public ChunkLock newChunkLock(@NonNls String desc) {
      */
     public void pulse() {
         List<GlowEntity> allEntities = new ArrayList<>(entityManager.getAll());
-        List<GlowPlayer> players = new LinkedList<>();
+        List<GlowPlayer> players = new ArrayList<>();
```

Comment:
Why was this changed? Perhaps `Collections.synchronizedList()` would be better?

Reply:
I used ArrayList because of O(1) access, which is required by ForkPulse and ForkPulsePlayers.
There is no need to use Collections.synchronizedList() because each entity is pulsed by only one thread anyway. There can't be memory consistency errors, at least I think so right now.